views/favorites/index.ejs

```diff
@@ -0,0 +1,10 @@
+<% include ../partials/header.ejs %>
+	<h1>Favorites</h1>
+	<% favorites.forEach(function(movie) { %>
+		<h2> <%= movie.Title %> </h2>
```

Comment:
Here you can only work with what information you have saved in your database. I would change it to something like this:
```
		<h2> <%= movie.Title %> </h2>
		<p><b>Year:</b> <%= movie.year %></p>
		<p><b>Poster:</b><img src="<%= movie.poster %>"></p>
		<p><b>IMDB ID:</b> <%= movie.imdbId %></p>
```